providers/prometheus/src/timeseries.rs

```diff
@@ -176,11 +176,12 @@ fn step_to_seconds(step: StepSize) -> u32 {
 }
 
 /// Returns the step to fetch from the given duration in seconds. We attempt
-/// to maintain roughly 30 steps for whatever the duration is, so that for a
-/// duration of one hour, we fetch per 2 minutes, and for a duration of one
-/// minute, we fetch per 2 seconds.
+/// to maintain roughly 120 steps for whatever the duration is, so that for a
+/// duration of one hour, we fetch per 30 seconds, however for a duration of one
+/// minute, we fetch per 1 seconds (as the step value is rounded up to a 
```

Comment:
I think the build error is just coming from an extra space at the end of this line, removing the trailing whitespace should unblock the pipeline